tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "skipLibCheck": true,
     "strict": true,
     "strictNullChecks": true,
-    "lib": ["esnext.array"],
+    "lib": ["ESNext"],
```

Comment:
I looked this up and seems like a good idea, we're going from a single specific esnext to the larger library, what motivated / forced this change?

Reply:
I was looking for another issue in the tsconfig and thought this would be simpler to maintain. I don't have any concern using ESNext features and no specific reason to limit it to just the array lib. So basically it is simplifying.